.github/workflows/deploy.yml

```diff
@@ -16,9 +16,7 @@ jobs:
           node-version: '12.x'
       - name: Test Build
         run: |
-          if [ -e yarn.lock ]; then
-          yarn install --frozen-lockfile
-          elif [ -e package-lock.json ]; then
+          if [ -e package-lock.json ]; then
```

Comment:
1. This check is unnecessary
2. `gh-release` still utilizes yarn (see [line 54](https://github.com/Drill4J/drill4j-website/blob/f60ff30f7c93833f30cc956b0841e48d6e370c90/.github/workflows/deploy.yml#L54))